src/styles/main.scss

```diff
@@ -0,0 +1,81 @@
+$cell-size: 100px;
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+}
```

Comment:
create a file for common styles and move it there
and move styles for `*` selector to this file as well